.github/workflows/benchmark-runtime-weights.yml

```diff
@@ -128,19 +128,19 @@ jobs:
             arg="\\$arg";
           fi
           for c in $chain; do
-            ssh -x -o StrictHostKeychecking=no "${{ steps.start_instance.outputs.remote_ip }}" -l ${{ env.BENCHMARK_SSH_USER }} -i ${{ env.BENCHMARK_SSH_KEYPATH }} 'bash -s' < scripts/benchmark-weight-remote.sh "$c" "${GITHUB_REF#refs/heads/}" "$arg"
+            ssh -x -o StrictHostKeychecking=no 172.31.14.5 -l ${{ env.BENCHMARK_SSH_USER }} 'bash -s' < scripts/benchmark-weight-remote.sh "$c" "${GITHUB_REF#refs/heads/}" "$arg"
```

Comment:
What's this IP? Can we hide it? Use GHA secrets if it's a fixed IP or put it somewhere on jumphost